app/assets/javascripts/templates/partials/contact.html.haml

```diff
@@ -1,7 +1,8 @@
 %div.contact-container
   %div.modal-centered{"ng-if" => "::enterprise.email_address || enterprise.website || enterprise.phone"}
     %p.modal-header {{'contact' | t}}
-    %p{"ng-if" => "::enterprise.phone", "ng-bind" => "::enterprise.phone"}
+    %p{"ng-if" => "enterprise.phone"}
+      %a{"ng-href" => "http://wa.me/{{::enterprise.phone}}", target: "_blank", "ng-bind" => "::enterprise.phone"}
```

Comment:
I would like to keep the old number, too, to enable people to call the number straight away instead of going to WhatsApp. Maybe we can add an additional link which indicates to the user that WhatsApp will be opened.

Reply:
I'd add a checkbox/radio button in the preference section: I guess that many users don't want to use WhatsApp at all.

Reply:
Hi @RachL, this is an example of what I had in mind for the frontend:
![whatsapp_example](https://user-images.githubusercontent.com/2237750/171257123-187746ed-744b-4561-814c-7f8b33c90422.png)
@mkllnk does that seem ok to you too?

As for the other platforms, I'm not sure, but I don't think there is a direct way to do that with a single link.